ReactWindows/ReactNative.Net46/Touch/TouchHandler.cs

```diff
@@ -0,0 +1,358 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace ReactNative.Touch
+{
+    class TouchHandler : IDisposable
+    {
+        private readonly FrameworkElement _view;
+        private readonly List<ReactPointer> _pointers;
+
+        private uint _pointerIDs;
+
+        public TouchHandler(FrameworkElement view)
+        {
+            _view = view;
+            _pointers = new List<ReactPointer>();
+
+            _view.MouseDown += OnPointerPressed;
```

Comment:
>OnPointerPressed [](start = 31, length = 16)

If you're keeping these in a separate implementation anyway, you can call these OnMouseDown, OnMouseMove, OnMouseUp, etc.